modules/programs/alacritty.nix

```diff
@@ -7,6 +7,26 @@
 let
   cfg = config.programs.alacritty;
   tomlFormat = pkgs.formats.toml { };
+
+  alacrittyTheme = cfg.themePackage.overrideAttrs (prevAttrs: {
```

Comment:
It's an interesting approach... only thing I don't like is that it forces every user of the module to have to build the package right? 

Would it be better as a `home.checks` type thing?